Makefile

```diff
@@ -1,10 +1,11 @@
-all     :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4' \
-           DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 \
-           dapp --use solc:0.6.12 build
-clean   :; dapp clean
-           # Usage example: make test match=SpellIsCast
-test    :; ./test-dssspell.sh $(match)
-deploy  :; make && dapp create DssSpell | xargs ./verify.py DssSpell
-estimate:; ./estimate-deploy-gas.sh
-flatten :; hevm flatten --source-file "src/DssSpell.sol" > out/flat.sol
-archive-spell :; ./archive-dssspell.sh $(date)
+all             :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4' \
+                    DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 \
+                    dapp --use solc:0.6.12 build
+clean           :; dapp clean
+                    # Usage example: make test match=SpellIsCast
+test            :; ./test-dssspell.sh $(match)
+test-dev        :; ./test-dev-dssspell.sh $(match)
```

Comment:
Rather than having two separate scripts here, what about one script that accepts an argument to enable dev mode?

That way when we make other changes as a result of tooling updates or whatever we don't need to reflect them in two separate files. In this case, the only real difference will be whether the optimizer is off or on.

Reply:
updated: https://github.com/makerdao/spells-mainnet/pull/210/commits/fdc1a0be5585a0a150e32fb6da2a86831c20dfe4

I left the test-dev command in there for shorthand, let me know if that bothers you